command/server/config/config.go

```diff
@@ -37,6 +38,8 @@ type Config struct {
 
 	ConcurrentRequestsDebug uint64 `json:"concurrent_requests_debug" yaml:"concurrent_requests_debug"`
 	WebSocketReadLimit      uint64 `json:"web_socket_read_limit" yaml:"web_socket_read_limit"`
+
+	RelayerTrackerPollInterval time.Duration `json:"relayer_tracker_poll_interval" yaml:"relayer_tracker_poll_interval"`
```

Comment:
Group it together with `Relayer` and `NumBlockConfirmations` fields, because it is related to relayer node(s). WDYT?

Also should this be `common.Duration` type maybe?

Reply:
I don't think this needs to be common.Duration. I am not even sure if this is even saved to some file.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1887/commits/f47c9ae2d3128f5fa8fa5bd00b49ef665ae38f18